vmaas/vulnerabilities.go

```diff
@@ -172,16 +173,15 @@ func (d *ProcessedDefinition) evaluate(
 	}
 }
 
+//nolint:funlen
 func (r *ProcessedRequest) processDefinitions(c *Cache, opts *options) (*ProcessedDefinitions, error) {
 	// Get CPEs for affected repos/content sets
 	// TODO: currently OVAL doesn't evaluate when there is not correct input repo list mapped to CPEs
 	//       there needs to be better fallback at least to guess correctly RHEL version,
 	//       use old VMaaS repo guessing?
 	candidateDefinitions := repos2definitions(c, r.OriginalRequest)
-	definitions := ProcessedDefinitions{
-		make(map[DefinitionID]ProcessedDefinition),
-		make(map[DefinitionID]ProcessedDefinition),
-	}
+	patchDefinitions := make(map[DefinitionID]ProcessedDefinition)
+	vulnerabilityDefinitions := make(map[DefinitionID]ProcessedDefinition)
 
 	for pkg, parsedNevra := range r.Packages {
```

Comment:
we can change `processedRequest.Packages` to `NevraString` type, this way it will be always slice, not map and we won't need to sort packages (hopefully)